src/Ledger/Gov.lagda

```diff
@@ -95,6 +95,18 @@ govActionPriority (ChangePParams _)        = 4
 govActionPriority (TreasuryWdrl _)         = 5
 govActionPriority Info                     = 6
 
+_gaPrRel_ : ℕ → ℕ → Type
+n gaPrRel m = (n ≡ m) ⊎ (n ≡ 0 × m ≡ 1) ⊎ (n ≡ 1 × m ≡ 0)
+
+_gaPrRel?_ : (n m : ℕ) → Dec (n gaPrRel m)
+n gaPrRel? m = (n ≟ m) ⊎-dec (n ≟ 0 ×-dec m ≟ 1) ⊎-dec (n ≟ 1 ×-dec m ≟ 0)
+
+_≡ᵍᵃ_ : GovAction → GovAction → Type
```

Comment:
I don't like `≡` here because it's not actually supposed to be an equality. I think `≈` is more appropriate.

Reply:
Agreed.  Will rename this `_≈_` (with no decoration since there's no conflict/ambiguity in `Gov` module).  Will also rename `_≡ᵍᵃ?_` to `_≈?_`.